src/main/java/org/proffart/football/training/api/impl/PlayerApiImpl.java

```diff
@@ -0,0 +1,46 @@
+package org.proffart.football.training.api.impl;
+
+import org.proffart.football.training.api.PlayerApi;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.*;
+
+/**
+ * Author Artak Mnatsakanyan
+ * Date 9/13/16
+ * Time 3:11 PM
+ */
+@Component
+@Path("/player")
+public class PlayerApiImpl implements PlayerApi {
+
+    @POST
+    @Override
+    public String createPlayer(String name) {
+        return null;
+    }
+
+    @PUT
+    @Override
+    public String savePlayer(String name) {
```

Comment:
Arguments must be same createPlayer method's arguments (there were some mistakes)
also you must to received player id from path 
@ Path("{id}") and get this id @PathParam("id") Integer id
```java
@PUT
@Path("{id}")
public String savePlayer(@PathParam("id") Integer id, String name) {
// ...
```